tests/e2e/api-core-tests/jest.config.js

```diff
@@ -0,0 +1,23 @@
+require('dotenv').config();
+const { BASE_URL, VERBOSE, USE_INDEX_PERMALINKS } = process.env;
+const verboseOutput = VERBOSE === 'true';
+
+// Update the API path if the `USE_INDEX_PERMALINKS` flag is set
+const useIndexPermalinks = USE_INDEX_PERMALINKS === 'true';
```

Comment:
For consistency with the [api package](https://github.com/woocommerce/woocommerce/blob/trunk/tests/e2e/api/src/http/http-client-factory.ts#L113:L117), index permalinks should enable pretty permalinks: `/wp-json/...`.

Reply:
Ah good call, updated! ✅